spec/models/manageiq/providers/openstack/cloud_manager/orchestration_template_hot_dialog_service_spec.rb

```diff
@@ -0,0 +1,95 @@
+describe ManageIQ::Providers::Openstack::CloudManager::OrchestrationTemplateHotDialogService do
+  let(:dialog_service) { described_class.new }
+  let(:template_hot)   { FactoryGirl.create(:orchestration_template_hot_with_content) }
+  let(:empty_template) { FactoryGirl.create(:orchestration_template_cfn) }
```

Comment:
I wonder if the `empty_template` test could be kept in [spec/services/orchestration_template_dialog_service_spec.rb](https://github.com/ManageIQ/manageiq/pull/12448/files#diff-a85f76dc0f83c55059a9a6e46f44ff05)? If not, I would suggest to move it to the Cfn related spec (it may be confusing when writing new service specs as OpenStack is one of the providers developers look for inspiration.